common.js

```diff
@@ -403,3 +405,33 @@ export function setupPath(newPathEntries) {
   core.addPath(newPath.join(path.delimiter))
   return msys2Type
 }
+
+export async function setupJavaHome() {
+  common.measure("Modifying JAVA_HOME for JRuby", async () => {
```

Comment:
```suggestion
  await measure("Modifying JAVA_HOME for JRuby", async () => {
```
Found the missing await